sources/packages/api/src/services/institution-location/institution-location.service.ts

```diff
@@ -3,7 +3,10 @@ import { RecordDataModelService } from "../../database/data.model.service";
 import { InstitutionLocation } from "../../database/entities/institution-location.model";
 import { Connection, UpdateResult } from "typeorm";
 import { ValidatedInstitutionLocation } from "../../types";
-import { InstitutionLocationTypeDto } from "../../route-controllers/institution-locations/models/institution-location.dto";
+import {
+  DesignatedAndNotDesignatedLocations,
```

Comment:
Not a part of this PR. The dto name DesignatedAndNotDesignatedLocations denotes locations apparently. So dto name could have been simple.

Reply:
Comment above updated. I thought initially that dto was not defined in this PR.

Reply:
DesignatedAndNotDesignatedLocations  is not a DTO here, also interface name updated